pkg/apis/sealed-secrets/v1alpha1/register.go

```diff
@@ -16,6 +16,8 @@ var (
 
 	// SchemeBuilder adds this group to scheme
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	// AddToScheme is a global function that registers this API group & version to a scheme
+	AddToScheme = SchemeBuilder.AddToScheme
```

Comment:
Is this used? Would it make sense to split it out to a different PR?

Reply:
it seems that github swallowed my answers.

k8s code-generator requires some boilerplate; this is now part of that boilerplate

Reply:
+1 thanks for checking